client/components/mma/identity/emailAndMarketing/NewsletterSection.tsx

```diff
@@ -82,6 +82,7 @@ export const NewsletterSection: FC<NewsletterSectionProps> = (props) => {
 	const { newsletters, clickHandler } = props;
 	return (
 		<PageSection
+			testSelector="your_newsletters"
```

Comment:
Is the `title` not enough for Cypress to find this? (I'm guessing not otherwise you wouldn't have added it - but interested in what the problem was!)

Reply:
I've replaced the data-cy attribute with a text and parent selector 👍